src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-form/rgw-storage-class-form.component.html

```diff
@@ -165,33 +135,42 @@
           <div cdsCol>
             <!-- Target Endpoint -->
             <cds-text-label
-            labelInputID="target_endpoint"
-            i18n
-            [invalid]="storageClassForm.showError('target_endpoint', formDir, 'invalidURL') || storageClassForm.showError('target_endpoint', formDir, 'required')"
-            [invalidText]="endpointError"
-            [helperText]="helpTextLabels.targetEndpointText"
-            >Target Endpoint
-            <input
-              cdsText
-              type="text"
-              placeholder="e.g, http://ceph-node-00.com:80"
-              i18n-placeholder
-              id="target_endpoint"
-              formControlName="target_endpoint"
-              [invalid]="storageClassForm.showError('target_endpoint', formDir, 'invalidURL') || storageClassForm.showError('target_endpoint', formDir, 'required')"
+              labelInputID="target_endpoint"
+              i18n
+              [invalid]="
+                storageClassForm.showError('target_endpoint', formDir, 'invalidURL') ||
+                storageClassForm.showError('target_endpoint', formDir, 'required')
+              "
               [invalidText]="endpointError"
-            />
-          </cds-text-label>
+              [helperText]="helpTextLabels.targetEndpointText"
+              >Target Endpoint
+              <input
+                cdsText
+                type="text"
+                placeholder="e.g, http://ceph-node-00.com:80"
```

Comment:
Placeholder is already a example
```suggestion
                placeholder="http://ceph-node-00.com:80"
```

Reply:
Please check on all places and remove it.